src/internal/components/chart-plot/focus-outline.tsx

```diff
@@ -1,20 +1,46 @@
 // Copyright Amazon.com, Inc. or its affiliates. All Rights Reserved.
 // SPDX-License-Identifier: Apache-2.0
-import React, { useEffect, useRef } from 'react';
+import React, { useEffect, useRef, useState } from 'react';
 
 import styles from './styles.css.js';
-import useFocusVisible from '../../hooks/focus-visible/index';
 import { Offset } from '../interfaces';
+import { isModifierKey } from '../../hooks/focus-visible';
 
 export interface FocusOutlineProps {
   elementKey?: null | string | number | boolean;
   elementRef?: React.RefObject<SVGSVGElement | SVGGElement>;
   offset?: Offset;
 }
 
+function useFocusVisibleState() {
```

Comment:
It is recommended to use css-only implementation for focus-visible (to allow migrate to native `:focus-visible` once our supported browsers allow that).

This is why I put this hook here to prevent reuses of this exceptional pattern

Reply:
I think this part can be refactored the same way as other components. I can do it myself as a follow-up.